crowdfunding_compassion/controllers/projects_controller.py

```diff
@@ -12,43 +12,36 @@
 
 
 class ProjectsController(Controller, FormControllerMixin):
-
-    @route('/projects', auth="public", website=True)
+    @route("/projects", auth="public", website=True)
     def get_projects_list(self, **kwargs):
-        values = {}
-        project_obj = request.env['crowdfunding.project']
-        values.update({
-            "project_list": project_obj.get_active_projects_list(number=9),
-        })
+        project_obj = request.env["crowdfunding.project"]
+
         return request.render(
-            "crowdfunding_compassion.project_list_view_template", values)
+            "crowdfunding_compassion.project_list_page",
+            {"projects": project_obj.sudo().get_active_projects(number=9)},
```

Comment:
Here you limit project number to 9 and there is no way to load more?
It should maybe have a start parameter to handle paginations or remove the 9 limit and put back the comment that you removed that points to the task of handling pagination:
```
TODO connect pagination to backend -> CO-3213
```